scripts/idfProjects1.csv

```diff
@@ -0,0 +1,9 @@
+https://github.com/wildfly/wildfly,b19048b72669fc0e96665b1b125dc1fda21f5993,naming,0
```

Comment:
Would be nice to have a header to describe each column, e.g., "#project,SHA,module,expectedFlakyTests"